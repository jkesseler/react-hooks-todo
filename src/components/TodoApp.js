import React from 'react';
import CssBaseline from '@material-ui/core/CssBaseline';

import Layout from './Layout';
import AddTodo from './AddTodo';
import TodoList from './TodoList';

const TodoApp = () => (
  <CssBaseline>
    <Layout>
      <AddTodo />
      <TodoList />
    </Layout>
  </CssBaseline>
);

export default TodoApp;
