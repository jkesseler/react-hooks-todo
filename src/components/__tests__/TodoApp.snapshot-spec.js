import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import unwrapMemo from '../../unwrap-memo';

import TodoApp from '../TodoApp';

describe('TodoApp', () => {
  it('renders without crashing', () => {
    const component = unwrapMemo(shallow, <TodoApp />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
