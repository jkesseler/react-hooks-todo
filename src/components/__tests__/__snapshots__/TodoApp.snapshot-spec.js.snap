// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoApp renders without crashing 1`] = `
<WithStyles(CssBaseline)>
  <Memo(Layout)>
    <Memo() />
    <Memo() />
  </Memo(Layout)>
</WithStyles(CssBaseline)>
`;
